urdf/src/model.cpp

```diff
@@ -177,6 +183,7 @@ bool Model::initString(const std::string & xml_string)
     this->root_link_ = model->root_link_;
     return true;
   }
+  fprintf(stderr, "Failed to parse robot description using: %s\n", best_plugin_name.c_str());
   return false;
```

Comment:
Small nit: I'd rearrange this logic so that `if (!model)`, we return early; otherwise, continue to the end of the method and `return true`.  I find that a bit more intuitive, but totally up to you whether you want to rearrange it that way.

Reply:
Return early on failure in 8bb27d1